### DISCUSSION TITLE
[求助委托]寻人启事

### DISCUSSION THREAD

**Discussion by zlf204028** [share121/inter-knot#2218] [其他]
![5](https://github.com/user-attachments/assets/b6f08207-0932-4fd0-abc4-262f5537bcb5)
姓名：无量塔姬子
性别：女
身高：167cm
体重：65kg（大概？）
年龄：30
特征：红色头发，喜欢喝酒
我的老师「无量塔姬子」失踪于2018-12-14 现在我还没找到她，希望绳匠门能帮我寻找一下。我相信我的她一定还活着，如果遭遇不测请联系我。
全身只有2800菲林了，找到后我愿意全部奉上！！

**Comment by MXR1-NJ** [comment]
这个委托我....好吧
我也想找到她，我追加2800菲林

**Comment by siyueershiwu** [comment]
好像莫名挨了一刀......

**Comment by guozichang2007** [comment]
我有线索了！推荐委托人可以去黑塔空间站奇物收藏里看看，疑似有委托人所寻找之人的武器（只不过好像已经碎了）

**Comment by Gjxkxjaka** [comment]
已经死了
![emotion_download_1723722604183](https://github.com/user-attachments/assets/40dfd148-97a6-496e-bed4-409775dd82fb)

**Comment by XokoukiguraX** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/05/31/919c110ea2f23f33557b2d630971193c_1208672549796794648.png" alt="姬子 计算" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/12/19/1223681a2483d22883bd7d9761020c09_5684454427826272188.png" alt="姬子 喝杯热水" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/17f60d2d36be5f75d00d88804df4a0c4.png" alt="姬子 微笑" width="75" >是她吗

**Comment by Phaethon-0019** [comment]
我……我找到她了……她，也是我的老师……我赶到那的时候，她已经……
![微信图片_20240815230528](https://github.com/user-attachments/assets/8abe9ff5-99c4-420c-8e1d-d28c364ba384)

**Comment by anything12121** [comment]
为什么这里也有刀子！？

**Comment by hua-yuanshuai** [comment]
不知道，但前阵子列车上的无名客有个长的很像，正巧他们在仙舟，我可以帮你问问景元<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/1719edbe458e6ea7147676632ddc0ca2_8989714708965482225.png" alt="景元 困" width="75" >
